react/src/components/buttons/SprkButton.js

```diff
@@ -36,6 +37,7 @@ const SprkButton = ({
       data-id={idString}
       data-analytics={analyticsString}
       disabled={disabled}
+      aria-label={loading ? spinningAriaLabel : undefined}
```

Comment:
This line is wiping out the initial value of aria-label when the component renders. It should only replace the value when loading is true.

Reply:
great catch, updated!